bitrise.yml

```diff
@@ -1431,6 +1431,27 @@ workflows:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
         machine_type_id: g2.8core
+  Bitrise-device-testing:
+    steps:
+    - cache-pull@2.1:
+        is_always_run: true
+    - xcode-build-for-test@2:
+        inputs:
+        - test_plan: Fennec_Enterprise_XCUITests
+        - scheme: Fennec_Enterprise_XCUITests
```

Comment:
I would also start trying with Fennec scheme...I see in xcode all the targets are automatically signed for Fennec, I'm not sure but that may help to avoid all these errors:
`/Users/[REDACTED]/git/Client.xcodeproj: error: No profiles for 'org.mozilla.ios.Fennec.WidgetKit' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'org.mozilla.ios.Fennec.WidgetKit'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'WidgetKitExtension' from project 'Client')`

Instead of trying to modify the project / workflows to fix signing issues..

Reply:
Fennec scheme also have the same test plans enabled, so it should work. It works locally to generate the xctesrun as: `Fennec_Smoketest3_iphoneos16.1-arm64.xctestrun`